ros_ethercat_model/include/ros_ethercat_model/joint.hpp

```diff
@@ -167,6 +168,9 @@ class JointState
   /// The effort the joint should apply in Nm or N (write-to variable)
   double commanded_effort_;
 
+  /// Indicates if commanded_effort_ is torque or PWM
+  sr_robot_msgs::ControlType::_control_type_type actuator_command_type_;
```

Comment:
I kind of dislike making the field a ROS msg when none of the others are...
I can't give a good reason for it though.
I guess storing the type could be done with an int enum, while a ROS msg class actually involves a much bigger object, with serialization methods and so on.

Reply:
`_control_type_type` is actually an alias for int enum. But I agree ideally we should get rid of existing dependency to `sr_robot_msgs` not add more usages.

Which one would you prefer locally declared enum with values: unspecified, torque, pwm or std::string with suggested values: "", "torque", "pwm" but allowing anything controller wants to pass to the driver?

Reply:
@guihomework thanks for suggestion at top level comment. We should be able to add desired field in child classes at `sr_core` repository instead of here.